### DISCUSSION TITLE
Which predicates should be used for citations used by a model?

### DISCUSSION THREAD

**Discussion by bilalshaikh42** [biosimulations/biosimulations#3982] [Q&A] [answered]
In the [metadata](https://docs.biosimulations.org/concepts/conventions/simulation-project-metadata/) is it possible to add references that the model used rather than just the citation  of the project itself? 

Some models may encode a pathway as laid out in a particular paper for example. The project should have a citation for this paper. 

Using successor does not seem to be appropriate here

**Comment by jonrkarr** [comment]
Currently, we use recommend using `http://biomodels.net/model-qualifiers/isDerivedFrom` to capture early model(s) that a model was derived from. This predicate could potentially be used to describe other resources that a model cites (i.e. similar to the references section of a paper which reported a model). This could be used to cite a principle that a model employs, which might be a little different from something that a model is derived from. Since these seem like two somewhat different types of relationships, I think we need to recommend another predicate for the latter.

**Reply by jonrkarr** [reply]
`http://purl.org/dc/terms/bibliographicCitation` could be a good fit. @bilalshaikh42 if this makes sense to you, we could convert this to an issue. In the mean time, this predicate can already be used by providing an `rdfs:label` for this predicate.

**Reply by bilalshaikh42** [reply]
http://purl.org/dc/terms/bibliographicCitation seems to be similar to what we currently have. The definition states "A bibliographic reference for the resource" which (to me) seems to specifically be how to cite the described resource rather than external sources.

http://purl.org/dc/terms/references seems to be the one we want as it goes in the correct direction. It is defined as: "A related resource that is referenced, cited, or otherwise pointed to by the described resource"

**Reply by jonrkarr** [reply]
Agreed references is better.

**Comment by bilalshaikh42** [comment, marked as answer]
http://purl.org/dc/terms/references seems to be the most relevant term to add
